packages/governance/src/contractGovernance/governApi.js

```diff
@@ -100,7 +100,7 @@ const setupApiGovernance = async (
         if (keyEQ(positive, outcome)) {
           assert(
             keyEQ(outcome, harden({ apiMethodName, methodArgs })),
-            `The question's method name (${apiMethodName}) and args (${methodArgs}) didn't match the outcome ${outcome}`,
+            X`The question's method name (${apiMethodName}) and args (${methodArgs}) didn't match the outcome ${outcome}`,
```

Comment:
I'll use this as an example of the opposite principle. In general, relatively static info like `apiMethodName` does not need to be redacted. By not redacting it, it is visible with much less trouble in more locations, further easing debugging. That's what `q` is for. Since `q` also quotes it, I removed the extra parens. 

```suggestion
            X`The question's method name ${q(apiMethodName)} and args (${methodArgs}) didn't match the outcome ${outcome}`,
```

Assumes
```js
const { details: X, quote: q } = assert;
```

Reply:
> That's what `q` is for.

I had misunderstood what it's for. Where is documentation on how to use `quote`? It's not mentioned in https://github.com/endojs/endo/blob/master/packages/ses/src/error/README.md  . With some searching I found https://github.com/endojs/endo/blob/db98546be8fda763daea27ac0513953200604ace/packages/ses/src/error/types.js#L166-L171 but that JSDoc isn't coming up for documenting the `quote` function uses. I suppose due to defining it as a callback type.

Reply:
My choices of declaring callbacks vs declaring in place were done in ignorance. Would be overjoyed to see them moved around to follow some useful principle.

Reply:
done